src/cluster/src/error.rs

```diff
@@ -44,6 +44,9 @@ pub enum K8InstallError {
     /// Encountered an error while performing one or more pre-checks
     #[error("Failed to perform one or more pre-checks")]
     PrecheckErrored(CheckResults),
+    /// Failed to update Fluvio cluster
+    #[error("Failed to update Fluvio")]
+    FailedUpdate,
```

Comment:
Should've this include why upgrade failed?

Reply:
Right now the only reason the `FailedUpdate` error would be given is if the platform version check times out. But I can add that to the description of the error.